test/rails_test_helper.rb

```diff
@@ -16,4 +18,7 @@ def self.skeleton(*path)
     skeleton("vendor", "cache", "gems", "example", "models"),
   ]
   config.root = skeleton(".")
+  config.logger = Logger.new(STDOUT)
 end
+
+T.cast(Dummy, Rails::Application).initialize!
```

Comment:
This assumes `Dummy` to be a value of type `Rails::Application`, but it is not - it is a subclass.